.github/pull_request_template.md

```diff
@@ -9,11 +9,22 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+If some steps are not applicable or in your judgment not necessary, remove and explain why rather than leave checkmarks blank.
+
+- [ ] I've tested the code as an end user would and included appropriate testing steps 
+- [ ] For all new functionality, I have added thorough, applicable tests that validate my changes
+- [ ] For improvements to existing functionality, I've followed the [campsite rule](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git), leaving it a little better than I found it. 
 - [ ] The added feature contains necessary observability
 - [ ] I have made corresponding changes to the documentation
 
+When done with the checklist, request a review from a [merger](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git) (list available [here](https://github.com/orgs/encodium/teams/mergers))
+
+## Reviewer checklist
+
+- [ ] Make sure the pre-review checklist has been completed.
+- [ ] I've run the code as an end user would.
```

Comment:
Is the assumption here that the developer includes necessary manual testing steps so that a reviewer can perform the same action?

How does this apply to QA engineers who manually test development stories?

Reply:
I'm pretty open to how you all think this should work. Do you have a suggestion for better language? 

When I've done reviews, I've typically had this as an expectation, but open to other approaches.

Reply:
What are current expectations for reviewers?

Reply:
We have this documented for new services here - https://github.com/encodium/.github-private/blob/main/profile/pages/engineering-principles.md#pull-requests

but we have not really ever fully addressed the disparity between this and the v1 platform.

Reply:
It also mentions automated testing, which greatly reduces the burden of a reviewer - https://github.com/encodium/.github-private/blob/main/profile/pages/engineering-principles.md#testing